app/controllers/results_controller.rb

```diff
@@ -333,8 +333,7 @@ def codeviewer
     # if dealing with a pdf file, get the number of images to display
     if @file.is_pdf?
```

Comment:
@RFrailich It seems this whole block can be removed, as it is calculating something which doesn't make any sense. Please also look for uses of the variable `@nb_pdf_files_to_download` and see if you can delete any of those.